Orange/widgets/visualize/owboxplot.py

```diff
@@ -322,6 +290,11 @@ def set_data(self, dataset):
 
         self.dataset = dataset
         if dataset:
+            sample_at = 1000
+            if len(self.dataset) > sample_at:
```

Comment:
I think this needs an additional condition: `isinstance(self.dataset, DaskTable)`. The current code would likely fail for big normal tables as `self.sample.compute()`.